### PR TITLE
Update framework readme - Closes #5827

### LINKED ISSUE #5827: Update framework readme
### Expected behavior
https://github.com/LiskHQ/lisk-sdk/blob/development/framework/README.md
should reflect latest architecture

### Actual behavior
Outdated

### Which version(s) does this affect? (Environment, OS, etc...)
5.0.0-alpha.1

### PR DESCRIPTION
**Pull request by shuse2**
### What was the problem?

This PR resolves #5827 

### How was it solved?

- Update image to reflect current architecture

### CODE REVIEW

File: framework/README.md [hunk 1]
```diff
@@ -6,19 +6,24 @@
 
 ## What is Lisk Framework
 
-Lisk Framework is an application framework responsible for establishing and maintaining the interactions between the modules of a Lisk blockchain application.
-
-Lisk Framework aims to provide a consistent and intuitive interface between each module and component. Currently, Lisk Framework establishes interactions between the `chain`, `api` and `network` modules.
+Lisk Framework is an application framework to create a blockchain application compatible with Lisk protocol.
```
Comment (ManuGowda) [inline framework/README.md:9]:
```suggestion
Lisk Framework is an application framework to create a blockchain application compatible with the Lisk protocol.
```

File: framework/README.md [hunk 2]
```diff
@@ -6,19 +6,24 @@
...
 
 ### Architecture Overview
 
-The architecture of Lisk Framework follows the research documented in [LIP0005](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0005.md). The diagram below provides a high-level overview of the architecture:
+The diagram below provides a high-level overview of the architecture:
 
 ![Logo](./docs/assets/diagram_framework.png)
 
-## Installation and usage
+The framework consists of 3 main parts, an application, a controller and a node.
```
Comment (ManuGowda) [inline framework/README.md:18]:
```suggestion
The framework consists of 3 main parts, an application, a controller, and a node.
```

File: framework/README.md [hunk 3]
```diff
@@ -6,19 +6,24 @@
...
+
+The application is an entry point and manages the controller and the node. The node is responsible for blockchain protocol logic using user defined modules, while the controller is responsible for maintaining plugins and managing interaction between plugins and the application.
```
Comment (ManuGowda) [inline framework/README.md:20]:
```suggestion
The application is an entry point and manages the controller and the node. The node is responsible for blockchain protocol logic using user defined modules, while the controller is responsible for maintaining plugins and managing the interaction between plugins and the application.
```

File: framework/README.md (hunk 2 above)
Comment (shuse2) [reply framework/README.md:18]: I think there is no need for `,` there

**Review by ManuGowda — APPROVED** [review]

**Review by mitsuaki-u — APPROVED** [review]

**Pull request merged**